README.md

```diff
@@ -310,10 +311,78 @@ This Guide *complements but does not replace* the advice you get from profession
 Working with the support of a professional can help you make better decisions when you
 have an understanding of the topic yourself, and know what questions to ask.
 
-## Fundamentals of Stock Corporations
+## Equity Compensation Basics
+
+### Compensation and equity
+
+🄳 **Compensation** is what you get in exchange for working for a company, including cash
+and non-cash pay.
+
+🄳 We say a person has **equity** in a company when they have some kind of ownership or
+likely future ownership in the company.
+
+Some compensation is cash—your salary and bonus, if you get one.
+Other parts are [benefits](https://en.wikipedia.org/wiki/Employee_benefits#United_States),
+like health insurance, family-related protections, perks, and retirement plans.
+But for many jobs in startups and established companies alike, compensation includes some
+form of ownership.
+
+🚧 Another term you may encounter is “total rewards”.
+Can we say more about this?
+
+🄳 [**Equity compensation**](http://www.investopedia.com/terms/e/equity-compensation.asp) is the
+practice of granting ownership in a company in exchange for work—that is, compensation in
+the form of equity.
 
 In this Guide we focus on equity compensation in stock corporations, the kind of company
 where ownership is represented by stock.
+We describe stock in [the next section](#fundamentals-of-stock-corporations). People are very
+rarely granted stock directly, with no strings attached.
+Instead, they’re given stock with additional restrictions placed on it, or are given
+contractual rights that later can lead to owning stock.
+These forms include restricted stock, stock options, and RSUs, each of which we’ll
+[describe in detail](#how-equity-is-granted).
```

Comment:
Very pleased with this! Introducing the topics ppl will hope to see and telling them when we'll get to them or how to jump to them, while making sure everyone is actually on the same page w/ what we are talking about when we talk about equity. This structure has both narrative sense and simple-to-complex foundational logic.